jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/PATCHSpec.scala

```diff
@@ -0,0 +1,77 @@
+package com.cerner.beadledom.jaxrs
+
+import com.cerner.beadledom.jaxrs.provider.{FakeModel, FakeResource}
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider
+
+import org.jboss.resteasy.core.Dispatcher
+import org.jboss.resteasy.mock.{MockDispatcherFactory, MockHttpRequest, MockHttpResponse}
+import org.jboss.resteasy.plugins.server.resourcefactory.POJOResourceFactory
+import org.scalatest._
+import org.scalatest.mock.MockitoSugar
+
+import java.io._
+
+import scala.collection.JavaConverters._
+import javax.ws.rs.core.MediaType
+
+/**
+  * Spec tests for {@link PATCH}.
+  *
+  * @author Eric Christensen
+  */
+class PATCHSpec extends FunSpec with BeforeAndAfterAll with ShouldMatchers with MockitoSugar {
+
+    val dispatcher: Dispatcher = MockDispatcherFactory.createDispatcher()
+    val noDefaults: POJOResourceFactory = new POJOResourceFactory(classOf[FakeResource])
+    dispatcher.getRegistry.addResourceFactory(noDefaults)
+    dispatcher.getProviderFactory.registerProvider(classOf[JacksonJsonProvider])
+
+    describe("PATCH") {
+      it("calls PATCH resource without error") {
+
+        val model = new FakeModel()
+            .setId("id")
+            .setName("name")
+            .setTimes(10)
+            .setTags(List.empty[String].asJava)
+            .setInnerModels(List.empty[FakeModel.FakeInnerModel].asJava)
+
+        val mapper = new ObjectMapper
+        val request: MockHttpRequest = MockHttpRequest.create("PATCH", "/fakeResource/Patch")
+        request.contentType(MediaType.APPLICATION_JSON)
+        request.content(new ByteArrayInputStream(mapper.writeValueAsBytes(model)))
+
+        val response: MockHttpResponse = new MockHttpResponse()
+        dispatcher.invoke(request, response)
+
+        response.getStatus shouldBe 200
+      }
+
+      it("should change the request model fields") {
```

Comment:
I would merge the two into one test. We are not testing two different aspects of the method here. Status code 200 means the request is served successfully. For PATCH a successful request mean that the resource is patched correctly. So, having these two checks in one test makes sense IMO.